python/triton/runtime/autotuner.py

```diff
@@ -154,8 +160,6 @@ def run(self, *args, **kwargs):
                 self.cache[key] = builtins.min(timings, key=timings.get)
                 self.pre_hook(args, reset_only=True)
                 self.configs_timings = timings
-                if self.verbose:
```

Comment:
Why do we need to remove verbose here?
Is there another way to print the best config?

Reply:
We have the function kernel.get_best_config() now. It seems like we did not even use the `verbose=true` at all

Reply:
I'd suggest leave `verbose=true` functionality there. I use it all the time to print best config.
And `get_best_config` can be mis-used when autotune is not present.

Reply:
> I'd suggest leave `verbose=true` functionality there. I use it all the time to print best config. And `get_best_config` can be mis-used when autotune is not present.

OK, you are using that. I just reverted the changes.